bridge/src/graphs/peg_out.rs

```diff
@@ -1897,6 +1898,30 @@ impl PegOutGraph {
     ) -> Result<Transaction, Error> {
         verify_if_not_mined(client, self.disprove_transaction.tx().compute_txid()).await?;
 
+        let assert_commit_1_txid = self.assert_commit_1_transaction.tx().compute_txid();
+        let assert_commit_2_txid = self.assert_commit_2_transaction.tx().compute_txid();
+        let onchain_assert_commit_1_tx = client
+            .get_tx(&assert_commit_1_txid)
+            .await
+            .map_err(Error::Esplora)?;
```

Comment:
[MINOR] These assignments can be rewritten using the `let`-`else` syntax. In this case we drop the `if` expressions below:
```suggestion
        let Some(onchain_assert_commit_1_tx) = client
            .get_tx(&assert_commit_1_txid)
            .await
            .map_err(Error::Esplora)?
        else {
            return Err(Error::Other(format!(
                "Esplora failed to retrieve a confirmed tx with id: {}",
                assert_commit_1_txid
            )));
        };
```

Reply:
Done